### PR TITLE
List ControllerManagerLeaderMigration feature gate in feature gates page

### PR DESCRIPTION
**Pull request by haardikdharma10**
Fixes: #28787

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -478,6 +479,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
   extended tokens by starting `kube-apiserver` with flag `--service-account-extend-token-expiration=false`.
   Check [Bound Service Account Tokens](https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/1205-bound-service-account-tokens/README.md)
    for more details.
+- `ControllerManagerLeaderMigration`: Enables Leader Migration for 
+  [kube-controller-manager](/docs/tasks/administer-cluster/controller-manager-leader-migration/#initial-leader-migration-configuration) and 
+  [cloud-controller-manager](/docs/tasks/administer-cluster/controller-manager-leader-migration/#deploy-cloud-controller-manager)
```
Comment (nckturner) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:484]: Can we just add: 

> which allows a cluster operator to live migrate controllers from the kube-controller-manager into an external controller-manager (e.g. the cloud-controller-manager) in an HA cluster without downtime.

Comment (haardikdharma10) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Thanks @nckturner :) Pushed the changes.

**Comment by jiahuif** [comment]
Thank you for the PR. The bigger issue would be missing feature gate in pkg/feature/kube_features.go
I've already open a PR to fix that. https://github.com/kubernetes/kubernetes/pull/103533

**Pull request merged**